ibverbs/src/lib.rs

```diff
@@ -256,6 +249,9 @@ impl<'devlist> Device<'devlist> {
     }
 }
 
+/// RDMA device attributes
+pub type DeviceAttr = ffi::ibv_device_attr;
```

Comment:
Should we wrap this type to provide nice accessors instead of just dumping out the `ffi` type directly?

Reply:
Sounds like a good idea, given that's what the other structs do :D